parlai/core/worlds.py

```diff
@@ -1499,6 +1533,622 @@ def shutdown(self):
         self.inner_world.shutdown()
 
 
+class QueueSignal(Enum):
+    """
+    Command signal used in the Queues for QueueWorld.
+
+    BATCH:                normal result; a batch is in the Queue
+    WORKER_FINISHED:      a worker is finished sending batch
+    RESET:                tell a worker to reset
+    RESET_METRICS:        tell a worker to reset_metrics
+    REPORT:               tell a worker to report
```

Comment:
Lordy. Do we need this with agent level metrics? Let’s coordinate

Reply:
unclear. i'll pull in the agent level metrics when it's merged and see what's necessary going from there